builder/common/run_config.go

```diff
@@ -680,11 +701,21 @@ func (c *RunConfig) Prepare(ctx *interpolate.Context) []error {
 		}
 	}
 
-	if c.Tenancy != "" &&
-		c.Tenancy != "default" &&
-		c.Tenancy != "dedicated" &&
-		c.Tenancy != "host" {
-		errs = append(errs, fmt.Errorf("Error: Unknown tenancy type %s", c.Tenancy))
+	var tenancy string
+	tenancies := []string{c.Placement.Tenancy, c.Tenancy}
```

Comment:
Here, I would fail if both are set. Otherwise, I think it could be nice to document the setting priority.